src/Sentry/Platforms/Cocoa/SentryOptions.cs

```diff
@@ -184,6 +184,15 @@ internal NativeOptions(SentryOptions options)
         /// </summary>
         public bool EnableTracing { get; set; } = false;
 
+        /// <summary>
+        /// Enable or disable the subscription to ObjCRuntime.Runtime.MarshalManagedException.
+        /// default value: true
+        /// </summary>
+        /// <remarks>
+        /// We recommend to set EnableMarshalManagedException to false for NativeAOT to avoid the used of unsupported marshelling behaviour
+        /// for more, see: https://github.com/getsentry/sentry-dotnet/pull/3729
+        /// </remarks>
```

Comment:
If we recommend that, shouldn't we do that by default on Native AOT?

Also it was enabled by default in .NET for a reason: https://github.com/xamarin/xamarin-macios/wiki/.NET-release-notes-Xcode-13.3#exception-marshalling-is-enabled-by-default

```suggestion
        /// <remarks>
        /// We recommend to set EnableMarshalManagedException to false for NativeAOT to avoid the used of unsupported marshelling behaviour.
        /// </remarks>
        /// <seealso href="https://github.com/xamarin/xamarin-macios/issues/15252#issuecomment-2349301905" />
```